### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dominic-ks**
Added examples of NCCO actions to README.md

## Description
 - Related issue: https://github.com/Vonage/vonage-php-sdk-core/issues/251
 - Added examples of NCCO Actions to the readme file

## Motivation and Context
 - Help php developers understand how to build a call with NCCO actions more quickly

## How Has This Been Tested?
 - All examples were tested using the php sdk before adding to this commit

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Additional documentation & examples

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes. - N/A
- [x] All new and existing tests passed. - N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -260,19 +260,148 @@ $outboundCall
...
+
+```php
+$outboundCall = new \Vonage\Voice\OutboundCall(
+    new \Vonage\Voice\Endpoint\Phone('14843331234'),
+    new \Vonage\Voice\Endpoint\Phone('14843335555')
+);
+
+$ncco = new NCCO();
+$record = new \Vonage\Voice\NCCO\Action\Record;
+$ncco->addAction($record->factory( array(
```
Comment (dragonmantank) [inline README.md:293]: Can we swap to using short array syntax here?

File: README.md [hunk 2]
```diff
@@ -260,19 +260,148 @@ $outboundCall
...
+```php
+$outboundCall = new \Vonage\Voice\OutboundCall(
+    new \Vonage\Voice\Endpoint\Phone('14843331234'),
+    new \Vonage\Voice\Endpoint\Phone('14843335555')
+);
+
+$ncco = new NCCO();
+
+$talk = new \Vonage\Voice\NCCO\Action\Talk;    
+$ncco->addAction($talk->factory( 'Please record your name.' , array(
```
Comment (dragonmantank) [inline README.md:365]: Can we swap to using short array syntax here?

File: README.md [hunk 3]
```diff
@@ -260,19 +260,148 @@ $outboundCall
...
+  'bargeIn' => true,
+)));
+
+$input = new \Vonage\Voice\NCCO\Action\Input;
+$ncco->addAction($input->factory( array(
```
Comment (dragonmantank) [inline README.md:370]: Can we swap to using short array syntax here?

File: README.md [hunk 4]
```diff
@@ -260,19 +260,148 @@ $outboundCall
...
+
+```php
+$outboundCall = new \Vonage\Voice\OutboundCall(
+    new \Vonage\Voice\Endpoint\Phone('14843331234'),
+    new \Vonage\Voice\Endpoint\Phone('14843335555')
+);
+
+$ncco = new NCCO();    
+$ncco->addAction(new \Vonage\Voice\NCCO\Action\Talk( 'We are just testing the notify function, you do not need to do anything.' ));
+$ncco->addAction(new \Vonage\Voice\NCCO\Action\Notify( array(
```
Comment (dragonmantank) [inline README.md:397]: Can we swap to using short array syntax here?

File: README.md [hunk 5]
```diff
@@ -260,19 +260,148 @@ $outboundCall
...
+### Send a text to voice call
 ```php
 $outboundCall = new \Vonage\Voice\OutboundCall(
     new \Vonage\Voice\Endpoint\Phone('14843331234'),
     new \Vonage\Voice\Endpoint\Phone('14843335555')
 );
+
 $ncco = new NCCO();
-$ncco->addAction(new \Vonage\Voice\NCCO\Action\Talk('This is a text to speech call from Vonage'));
+$ncco->addAction(new \Vonage\Voice\NCCO\Action\Talk( 'This is a text to speech call from Nexmo' ));
```
Comment (dragonmantank) [inline README.md:332]: I'd like to keep the string parameters consistent, so can we remove the extra whitespace here?

File: README.md [hunk 6]
```diff
@@ -260,19 +260,148 @@ $outboundCall
...
+
+### Stream an audio file on a call
+```php
+$outboundCall = new \Vonage\Voice\OutboundCall(
+    new \Vonage\Voice\Endpoint\Phone('14843331234'),
+    new \Vonage\Voice\Endpoint\Phone('14843335555')
+);
+
+$ncco = new NCCO();
+$ncco->addAction(new \Vonage\Voice\NCCO\Action\Stream( 'https://my-mp3.url' ));
```
Comment (dragonmantank) [inline README.md:346]: I'd like to keep the string parameters consistent, so can we remove the extra whitespace here?

File: README.md (hunk 2 above)
Comment (dragonmantank) [inline README.md:365]: I'd like to keep the string parameters consistent, so can we remove the extra whitespace here?

File: README.md [hunk 7]
```diff
@@ -260,19 +260,148 @@ $outboundCall
...
+### Send a notification to a webhook url
+
+```php
+$outboundCall = new \Vonage\Voice\OutboundCall(
+    new \Vonage\Voice\Endpoint\Phone('14843331234'),
+    new \Vonage\Voice\Endpoint\Phone('14843335555')
+);
+
+$ncco = new NCCO();    
+$ncco->addAction(new \Vonage\Voice\NCCO\Action\Talk( 'We are just testing the notify function, you do not need to do anything.' ));
```
Comment (dragonmantank) [inline README.md:396]: I'd like to keep the string parameters consistent, so can we remove the extra whitespace here?

**Review by dragonmantank — CHANGES_REQUESTED** [review]
Overall this looks good, just have a few formatting changes.

Thanks for this!

**Comment by dominic-ks** [comment]
Thanks for the comments @dragonmantank I believe I've addressed all of those.

**Pull request merged**